addon/components/flexberry-checkbox.js

```diff
@@ -2,78 +2,112 @@
   @module ember-flexberry
 */
 
-import Ember from 'ember';
-import FlexberryBaseComponent from './flexberry-base-component';
+import FlexberryBaseComponent from 'ember-flexberry/components/flexberry-base-component';
 
 /**
-  Checkbox component for Semantic UI.
+  Flexberry checkbox component with [Semantic UI checkbox](http://semantic-ui.com/modules/checkbox.html) style.
 
-  @example
-    ```handlebars
-    {{flexberry-checkbox checked=model.enabled label='Enabled'}}
-    ```
+  Usage:
```

Comment:
For correct display we must have @example. "The code example to be displayed should be indented on a new line directly underneath the @example tag."